README.md

```diff
@@ -46,7 +46,7 @@ It takes time to build up collection of tools used in CTF and remember them all.
 
 *Tools used for creating CTF challenges*
 
-- [Kali Linux CTF Blueprints](https://doc.lagout.org/security/Packt.Kali.Linux.CTF.Blueprints.Jul.2014.ISBN.1783985984.pdf) - Online book on building, testing, and customizing your own Capture the Flag challenges.
+- [Kali Linux CTF Blueprints](https://www.packtpub.com/eu/networking-and-servers/kali-linux-ctf-blueprints) / [download link](https://mega.nz/#!ijQgQKTY!EvhUfLAm7hvV4eLGEVnRRp-lzn7f7JpmMGEZzf-_Kn0) - Online book on building, testing, and customizing your own Capture the Flag challenges.
```

Comment:
This seems like a pirated version. Unsure if we should add this to the repo. Thoughts @apsdehal ?

Reply:
I agree with @DhavalKapil I added the pdf download simply because it used to have it before. 
I'd rather we do not include the download link.

Reply:
Sounds good, can you remove it from the PR?

Reply:
Removed it and also resolved the conflict